scripts/gendocs/generate

```diff
@@ -163,32 +181,36 @@ genversionwithcli "release-1.7" "v1.7-docs"
 
 # Special handling for release-1.6 because the legacy APIs were still present in
 # the CRDs and in public `pkg/` directory but we do not want to generate API
-# docs for those legacy API docs.
+# docs for those legacy API versions.
+
 # In cert-manager 1.6 cert-manager.io and acme.cert-manager.io alpha and beta
 # API versions are no longer served, but the apis are still in the codebase. We
 # don't want to show the docs for those API versions so this is a workaround.
 # This will only be necessary for the release-1.6 branch.
 
-#checkout "release-1.6"
-#
-## Generate CLI reference docs before deleting the legacy API,
-## as they will not compile.
-#genclireference "v1.6-docs" "cmd/acmesolver" "acmesolver"
-#genclireference "v1.6-docs" "cmd/cainjector" "cainjector"
-#genclireference "v1.6-docs" "cmd/ctl" "cmctl"
-#genclireference "v1.6-docs" "cmd/controller" "controller"
-#genclireference "v1.6-docs" "cmd/webhook" "webhook"
-#
-#rm -r pkg/apis/acme/v1alpha2
-#rm -r pkg/apis/acme/v1alpha3
-#rm -r pkg/apis/acme/v1beta1
-#rm -r pkg/apis/certmanager/v1alpha2
-#rm -r pkg/apis/certmanager/v1alpha3
-#rm -r pkg/apis/certmanager/v1beta1
-#gendocs "v1.6-docs"
-# v1.6 end
-
+generate_release16() {
+    # The special-case for release-1.6 is in a function to make it easier to
+    # comment and uncomment below
+    checkout "release-1.6"
+    # Generate CLI reference docs before deleting the legacy API,
+    # as they will not compile otherwise
+    genclireference "v1.6-docs" "cmd/acmesolver" "acmesolver"
+    genclireference "v1.6-docs" "cmd/cainjector" "cainjector"
+    genclireference "v1.6-docs" "cmd/ctl" "cmctl"
+    genclireference "v1.6-docs" "cmd/controller" "controller"
+    genclireference "v1.6-docs" "cmd/webhook" "webhook"
+
+    rm -r pkg/apis/acme/v1alpha2
+    rm -r pkg/apis/acme/v1alpha3
+    rm -r pkg/apis/acme/v1beta1
+    rm -r pkg/apis/certmanager/v1alpha2
+    rm -r pkg/apis/certmanager/v1alpha3
+    rm -r pkg/apis/certmanager/v1beta1
+
+    gendocs "v1.6-docs"
+}
 
+#generate_release16
```

Comment:
note: this simply exists to make it easier to comment and uncomment the generation of release-1.6